search/serializers.py

```diff
@@ -86,6 +87,48 @@ def postprocess_fields(self, discussions_obj, serialized_data):
         return {"author_channel_membership": sorted(get_channels(discussions_obj.user))}
 
 
+class ESCourseSerializer(ESSerializer):
+    """
+    Elasticsearch serializer class for courses
+    """
+
+    object_type = COURSE_TYPE
+    use_keys = [
+        "course_id",
+        "short_description",
+        "full_description",
+        "platform",
+        "language",
+        "semester",
+        "year",
+        "level",
+        "start_date",
+        "end_date",
+        "enrollment_start",
+        "enrollment_end",
+    ]
+    rename_keys = {"title": "course_title", "image_src": "course_image"}
+
+    @property
+    def base_serializer(self):
+        from course_catalog.serializers import CourseSerializer
+
+        return CourseSerializer
+
+    def postprocess_fields(self, discussions_obj, serialized_data):
+        return {
+            "instructors": [
+                "{} {}".format(i.first_name, i.last_name)
+                for i in discussions_obj.instructors.iterator()
+            ],
+            "topics": [t.name for t in discussions_obj.topics.iterator()],
+            "prices": [
```

Comment:
`"prices": discussions_obj.prices.values("price", "mode")` gives you the same result in less code.